src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/UpdateVCFSequenceDictionary.java

```diff
@@ -24,6 +24,36 @@
 /**
  * Updates the sequence dictionary in a variant file using the dictionary from a variant, alignment, reference,
  * or dictionary file.
+ *
+ * <p>This tool is designed to update the sequence dictionary in a variant file using a dictionary from another variant,
+ * alignment, dictionary, or reference file. The dictionary must be valid, i.e. must contain a sequence record, for all
+ * variants in the target file.
+ * </p>
+ *
+ * <h3>Example</h3>
+ *
+ * <pre>
+ * gatk UpdateVCFSequenceDictionary \
+ *   -O updated_sequence_dictionary.dict \
+ *   -V variants.vcf \
+ *   --source-dictionary input_dictionary.dict \
+ *   --replace=true
+ * </pre>
+ *
+ * <pre>
+ * gatk UpdateVCFSequenceDictionary \
+ *   -O updated_sequence_dictionary.dict \
+ *   -R reference.fa \
+ *   -V variants.vcf \
+ *   --replace=true
+ * </pre>
```

Comment:
- Example --> Usage examples
- Since we strive to have default parameters reflect the most common use case, and I see you're one step ahead and added an explanation of the `--replace=true` option below. We need to illustrate within the example command as well the optional nature of the parameter.
- The `--source-dictionary` is optional if we provide the reference
- For whatever reason, I am getting an error with `-O` but not with `--output`:
```
***********************************************************************

A USER ERROR has occurred: O is not a recognized option

***********************************************************************
```

Looks like a bug that perhaps you are able to fix? If not, we need to put in an issue ticket. In the meanwhile, we can specify the parameter that we know works in the example commands:

Use the contig dictionary from a BAM (SQ lines) to replace an existing dictionary in the header of a VCF. 
<pre>
gatk UpdateVCFSequenceDictionary \
    -V cohort.vcf.gz \
    --source-dictionary sample.bam \
    --output cohort_replacedcontiglines.vcf.gz \
    --replace=true
</pre>

Use a reference dictionary to add reference contig lines to a VCF without any.
<pre>
gatk UpdateVCFSequenceDictionary \
    -V resource.vcf.gz \
    --source-dictionary reference.dict \
    --output resource_newcontiglines.vcf.gz 
</pre>

Use the reference set to add contig lines to a VCF without any.
<pre>
gatk UpdateVCFSequenceDictionary \
    -V resource.vcf.gz \
    -R reference.fasta \
    --output resource_newcontiglines.vcf.gz 
</pre>

Reply:
Thanks for catching the error! I think I have fixed the -O option problem in my branch. Did you run into this problem on my branch or on master?

Reply:
Updated the docs as you suggested